cross/ds.evansport/Makefile

```diff
@@ -0,0 +1,35 @@
+PKG_NAME = ds.evansport
+PKG_VERS = 6.1.dev
+PKG_EXT = txz
+PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://sourceforge.net/projects/dsgpl/files/toolkit/DSM6.1
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+
+DEPENDS = 
+REQUIRED_DSM = 6.1
+UNSUPPORTED_ARCHS = $(ARM_ARCHES) $(PPC_ARCHES) $(x64_ARCHES)
+
+HOMEPAGE = https://en.wikipedia.org/wiki/List_of_Intel_Atom_microprocessors
+COMMENT  = Synoloy evansport toolkit including headers & libraries for GPU acceleration based on the PowerVR SGX545 from Imagination Technologies
+LICENSE  = unknown
+
+EXTRACT_TARGET = ds.evansport_custom_extract
+CONFIGURE_TARGET = ds.evansport_custom_configure
+COMPILE_TARGET = ds.evansport_custom_compile
+INSTALL_TARGET = ds.evansport_custom_install
```

Comment:
These targets are not defined. Please use `none`.

Reply:
Done.
You had also mentioned previously to perhaps move this out from `cross` directory.
Any new thoughts on this or should this be looked into at some other time?